scripts/run-singularity

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
```

Comment:
Is this an original file coming from 2.X ?

Reply:
+1